Android.mk

```diff
@@ -56,6 +57,19 @@ ifeq ($(CFG_SECURE_DATA_PATH),y)
 srcs += sdp_basic.c
 endif
 
+define my-embed-file
+$(TARGET_OUT_HEADERS)/$(1).h: $(2)
```

Comment:
Where's `TARGET_OUT_HEADERS` defined? Maybe `$(LOCAL_PATH)/host/xtest`?

@liuyq please comment as necessary

Reply:
I think it's defined by the build system